dbms/src/Server/FlashGrpcServerHolder.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+#include <Common/assert_cast.h>
+#include <Debug/astToExecutor.h>
+#include <Flash/DiagnosticsService.h>
+#include <Flash/FlashService.h>
+#include <Flash/Mpp/GRPCCompletionQueuePool.h>
+#include <Server/RaftConfigParser.h>
+
+namespace DB
+{
+namespace ErrorCodes
+{
+extern const int IP_ADDRESS_NOT_ALLOWED;
+} // namespace ErrorCodes
+
+void handleRpcs(grpc::ServerCompletionQueue * curcq, const LoggerPtr & log);
```

Comment:
Is this function referenced by other c++ files?

Reply:
I have moved it.